src/ScrumBoardItBundle/Services/AbstractApi.php

```diff
@@ -64,6 +68,44 @@ protected function call($url)
         return json_decode($content);
     }
 
+    protected function send($url, $content, $nbArguments)
+    {   
+        $ch = curl_init();
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_HTTPHEADER, [
+            'Authorization: Basic ' . $this->getUser()->getHash() . "\naccept: application/json
+            \naccept-language: en-US,en;q=0.8
+            \nuser-agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/49.0.2623.108 Chrome/49.0.2623.108 Safari/537.36"
```

Comment:
Please split the headers with commas instead of carriage returns.
In addition, only "Authorization: Basic [...]" and "Accept: application/vnd.github.v3+json" are usefull